e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -5,7 +5,10 @@ import { login } from '../../utils/login';
 
 const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
 
-describeOnCondition(edition === 'EE')('Releases page', () => {
+/**
+ * Skip tests on v5 until Releases + Scheduling are migrated to v5
+ */
+describeOnCondition(/*edition === 'EE'*/ false)('Releases page', () => {
```

Comment:
These tests do pass on `main` though, why don't they pass now?

Reply:
This was a mistake, they should pass here too. Thanks and good catch.